src/core/AnimatedNode.js

```diff
@@ -2,6 +2,8 @@ import ReanimatedModule from '../ReanimatedModule';
 
 const UPDATED_NODES = [];
 
+export const initializedNodes = new Set();
```

Comment:
I suggest that we keep this object internal to the current module and only export a method returning its size.

We can call it:
```
getNumberOfInitializedNodes_onlyForTests()
```